.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,12 @@
-* [ ] Put an X in this bracket to confirm you have read the
-  [External Contribution Guidelines](https://github.com/leanprover/lean4/blob/master/doc/contributions.md).
+# Before submitting
 
-* Please put the link to your `RFC` or `bug` issue here.
-  PRs missing this link will be marked as `missing RFC`.
+* Ensure your PR follows the [External Contribution Guidelines](https://github.com/leanprover/lean4/blob/master/CONTRIBUTING.md).
+* Add the link to your `RFC` or `bug` issue below.
+* If the issue does not already have approval from a developer: submit the PR as draft.
+* Remove the whole "Before submitting" section.
 
-* If that issue does not already have approval from a developer,
-  please be sure to open this PR in "Draft" mode.
+P.S. You can manage the `awaiting-review`, `awaiting-author`, and `WIP` labels yourself, by writing a comment containing one of these labels on its own line.
 
-* Please make sure the PR has excellent documentation and tests.
-  If we label it `missing documentation` or `missing tests` then it needs fixing!
```

Comment:
Where did this go?

Reply:
* "Please make sure the PR has excellent documentation and tests" - it's in the "External Contribution Guidelines", mentioned on the first line. Should we emphasize it again here?
* "If we label it `missing documentation` or `missing tests` then it needs fixing!" - seems obvious, should we state it explicitly?